packages/toolbox/src/swarm.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import os from 'os'
+import keythereum from 'keythereum'
+import fs from 'fs-extra'
+
+export type KeyObject = {
+  address: string,
+  Crypto: {
+    cipher: string,
+    ciphertext: string,
+    cipherparams: {
+      iv: string,
+    },
+    mac: string,
+    kdf: string,
+    kdfparams: {
+      c: number,
+      dklen: number,
+      prf: string,
+      salt: string,
+    },
+  },
+  id: string,
+  version: number,
+}
+
+const homedir = os.homedir()
+const password = 'mainframe'
+const datadir = `${homedir}/.ethereum`
+
+export const createKeyStore = async (): Promise<string> => {
+  const params = { keyBytes: 32, ivBytes: 16 }
+  const dk = keythereum.create(params)
+  const kdf = 'pbkdf2'
+
+  const options = {
+    kdf: kdf,
+    cipher: 'aes-128-ctr',
+    kdfparams: {
+      c: 262144,
+      dklen: 32,
+      prf: 'hmac-sha256',
+    },
+  }
+
+  const keyObject = keythereum.dump(
+    password,
+    dk.privateKey,
+    dk.salt,
+    dk.iv,
+    options,
+  )
+  await fs.ensureDir('keystore')
+  const path = await keythereum.exportToFile(keyObject)
+  await fs.ensureDir(datadir)
+  await fs.copy(path, `${datadir}/${path}`)
+  return keyObject.address
+}
+
+export const importKeyStore = async (address: string): Promise<KeyObject> => {
+  return keythereum.importFromFile(address, datadir)
+}
+
+export const getPrivateKey = async (
+  address: string,
+  encoding: string,
```

Comment:
Maybe we can make this second argument optional by setting a default value, `encoding?: string = 'hex'`